.github/workflows/java.yaml

```diff
@@ -14,30 +14,21 @@ on:
 jobs:
   tests-java-1_8:
     name: Test
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-24.04
+    container: ubuntu:20.04
```

Comment:
These java tests require an old java version, so we still run ubuntu 20.04 in a container